website/docs/implementations/Software.md

```diff
@@ -1,6 +1,46 @@
 ---
-sidebar_position: 25
+sidebar_position: 30
 title: Software Solutions
 ---
 
+Please refer to [Implementation Register](index.md) for general information about the purpose, scope, and process of registering UNTP implementations.
+
+
+## Implementation Register
+
+Summary list of software products and status of implementation.  Scope defines the list of UNTP specifications supported. 
+
+
+|Organisation Name |Product Name|UNTP Scope|Status|
+|--|--|--|--|
+|[UN/CEFACT](#uncefact)|UNTP Reference Implementations|VCP, DPP, DTE, DCC, DFR, DIA, IDR|planned| 
+| | | | |
+
+
+## Implementation Details
+
+
+### UN/CEFACT
+
+** Organisation Information **
+
+|Logo|Name|Implementation Statement|Registration Date|Geographic scope|
+|--|--|--|--|--|
+|![uncefact](../../implementations/uncefact/logo.png)|[UN/CEFACT](https://unece.org/trade/uncefact)|Supporting UNTP implementers with a reference implementation suite comprising open source software and conformity testing tools.|01-Sept-2024|Global|
+
+** Product Information **
+
+A complete suite of reference implementations and test tools will be available at the same time as UNTP version 0.5.0 pilot testing release (due 1st Oct 2024).
+
+|Product Name|Description|Product Version(s)|UNTP Scope|UNTP Version(s)|Implementation Date|Test Report|
+|--|--|--|--|--|--|
+|[VCkit](https://github.com/uncefact/project-vckit)|Verifiable credential issuing and verifying toolkit|v0.5.0 |VCP|v0.5.0|01-Oct-2024|TBA |
```

Comment:
Would it be a better experience not to have entries listed already, but add these through the registration process?

Reply:
Yes good idea.  Even though it's the UN reference implementation, we should still eat our own dog food and push it through the registration process.  I've modified the software page to just have a generic "sample" entry like the others.